File: JS/NossosDiasNuncaVoltarao.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
 const TEST = require('tape')
 const OBJ = require('./NossosDiasNuncaVoltarao.js')
 
 TEST('Starting NossosDiasNuncaVoltarao test...', (t) => {
     //Test 1
     t.assert(OBJ.cutWord(1) === "L", "Expect: 1 = L")
     //Test 2
     t.assert(OBJ.cutWord(3) === "LIF", "Expect: 3 = LIF")
     //Test 3
     t.assert(OBJ.cutWord(7) === "LIFE IS", "Expect: 7 = LIFE IS")
 
     t.end()
 })
 

File: GO/NossosDiasNuncaVoltarao_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestNossosDiasNuncaVoltarao(t *testing.T) {
	tables := []struct {
		pos  int
		want string
	}{
		{1, "L"},
		{3, "LIF"},
		{7, "LIFE IS"},
	}

	for _, table := range tables {
		got := cutWord(table.pos)

		if got != table.want {
			t.Errorf("Waiting for this %s but the return was this: %s", table.want, got)
		}
	}
}
